apps/api/v2/src/modules/webhooks/outputs/oauth-client-webhook.output.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Expose, Type } from "class-transformer";
+import { IsInt, IsEnum, ValidateNested } from "class-validator";
+
+import { SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
+
+import { WebhookOutputDto } from "./webhook.output";
+
+export class OAuthClientWebhookOutputDto extends WebhookOutputDto {
+  @IsInt()
+  @Expose()
+  readonly oAuthClientId!: string;
+}
+
+export class OAuthClientWebhookOutputResponseDto {
+  @Expose()
+  @ApiProperty({ example: SUCCESS_STATUS, enum: [SUCCESS_STATUS, ERROR_STATUS] })
+  @IsEnum([SUCCESS_STATUS, ERROR_STATUS])
+  status!: typeof SUCCESS_STATUS | typeof ERROR_STATUS;
```

Comment:
FYI: In `packages/platform/types/api.ts` we have defined `ApiResponseWithoutData` that can be extended:
```
export class GetTeamEventTypeOutput extends ApiResponseWithoutData {
  @ValidateNested()
  @Type(() => TeamEventTypeOutput_2024_06_14)
  data!: TeamEventTypeOutput_2024_06_14;
}
```

so we don't re-define status manually.